ic-agent/src/agent/response_authentication.rs

```diff
@@ -147,14 +145,19 @@ pub(crate) fn lookup_reply(
     Ok(RequestStatusResponse::Replied { reply })
 }
 
-pub fn lookup_value<'a>(
+pub fn lookup_value<'a, P>(
```

Comment:
Would you please add an entry to the changelog since it looks like technically this is a breaking change?